tools/cli/go-whisk-cli/commands/package.go

```diff
@@ -395,15 +395,15 @@ var packageListCmd = &cobra.Command{
         return entityNameError(qualifiedName.GetEntityName())
       }
 
-      client.Namespace = qualifiedName.GetNamespace()
+      Client.Namespace = qualifiedName.GetNamespace()
     }
 
     options := &whisk.PackageListOptions{
       Skip:   flags.common.skip,
       Limit:  flags.common.limit,
     }
 
-    packages, _, err := client.Packages.List(options)
+    packages, _, err := Client.Packages.List(options)
```

Comment:
Debug message below does not match. Should be `Client.Packages.List` instead of `client.Packages.List`.